### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by TheEnthusiasticAs**

### CODE REVIEW

**Comment by w1g0** [comment]
Can you add that the Mantis Client isn't fully operational to avoid questions why people can't sync with Magneto?

**Comment by TheEnthusiasticAs** [comment]
@w1g0 thx for the comments. I updated the PRs

**Review by w1g0 — APPROVED** [review]

**Review by developerkevin — CHANGES_REQUESTED** [review]
as far as placing mantis on the list of clients, I think we should either hold off on that until its live on ETC mainnet or add the word BETA next to it, and make sure to make it apparent that it is not mainnet ready until Magneto opcodes implemented but even then there may be other upgrades to add right after i.e. Keccak, FlyClients or NiPoPoWs, Account Versioning, others.

File: content/development/clients/clients.en.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ items:
     key: Mantis
     name: Mantis
     link: https://mantisclient.io/
-    description: Mantis will be a fully featured client and wallet UI, developed from the ground up and written in Scala for the Ethereum Classic network (ETC). Created by IOHK to add robustness and variety to the ETC's client share, it includes optimizations and network upgrades that improve network security, sustainability, and performance in the long term. Mantis should be considered as Beta software and is not compatible with mainnet or any testnet besides Sagano.
+    description: Mantis will be a fully featured client and wallet UI, developed from the ground up and written in Scala for the Ethereum Classic network (ETC). Created by IOHK to add robustness and variety to the ETC's client share, it includes optimizations and network upgrades that improve network security, sustainability, and performance in the long term. Mantis should be considered as Beta software and is not compatible with mainnet or any testnet besides Sagano. Also, isn't updated with the Magneto version at the moment.
```
[the remark this answers is not in the document]
Comment (w1g0) [reply content/development/clients/clients.en.yaml:38]: Stating explicitly in the in the name that Mantis is not main net ready/Beta version is a good idea. If people want to configure the client to sync with Sagano they should be able to with the provided link.

**Review by hitchcott — APPROVED** [review]

File: content/development/clients/clients.en.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (hitchcott) [reply content/development/clients/clients.en.yaml]: I added those changes

**Review by hitchcott — APPROVED** [review]

**Review by developerkevin — APPROVED** [review]
LGTM

**Pull request merged**